docs/document-similarity.md

```diff
@@ -0,0 +1,11 @@
+Document Similarity is a method to determine the similarity between certain sentences,
+paragraphs, and documents.
+It is widely used in identifying plagiarisms.
+It means that the program converts a document into a collection of strings and vectors.
+Similar words or sentences will be marked as a vector and compared to the other document.
+With this, we can calculate the angle between the document vectors.
+The vector angle should be from 0-90 which demonstrates how similar they are.
+If the vector angle is close to 0, it is really similar.
+If the vector angle is close to 90, documents are completely different.
```

Comment:
I think this statement makes the graph more confusing because, in the graph, the values range between what it looks like 0-1 and the higher they are to more similar the document it. I think the focus of the description should be to help the user understand the graph and some of the background behind calculating the data but this might be unnecessary here. We can discuss this further if you have questions on how to change it